ooni/api
# Add support for returning probe_asn and probe_ip as part of check-in

~TODO~ DONE:
* [x] Figure out how to test the feature
* [x] Figure out how we want to ship database updates and ensure they are fresh

For the second point I think we either:
* Add the download and ensuring that they are fresh as part of the flask app 
* Add logic for downloading and periodically checking for updates inside of the debian package

Since it's pretty critical that we keep the DBs fresh, I think this should be automated as much as possible
